docs/errors/E0373.md

```diff
@@ -0,0 +1,29 @@
+# E0373: Pointless Comparison Against String Literal
+
+A condition with a string literal will always evaluate to true
+
+```typescript
+if("x" || "y"){
+    let x = y;
+}
+```
+
+To fix this warning, remove the condition.
+
+```typescript
+let x = y;
+```
+
+Or, 
+
+```typescript
+case "x" || "y":
+```
```

Comment:
Must fix: Examples with bad code belong in the first code block. Also, they must be syntactically valid aside from the diagnostic, so you must include a `switch` statement.